internal/globalconfig/globalconfig.go

```diff
@@ -62,3 +64,27 @@ func RuntimeID() string {
 	defer cfg.mu.RUnlock()
 	return cfg.runtimeID
 }
+
+// SetHeaderTag adds config for header `from` with tag value `to`
+func SetHeaderTag(from, to string) {
+	cfg.headersAsTags[from] = to
+}
+
+// GetHeaderTag returns tag representation of header `header`
+func GetHeaderTag(header string) (tag string) {
+	return cfg.headersAsTags[header]
+}
+
```

Comment:
This is only used in tests, correct? Is it necessary to have it part of the API in this way, or can this just be directly in the tests where it's needed?

Reply:
Just to make sure we're on the same page, having a public getter for the config can be a problem due to this: https://github.com/DataDog/dd-trace-go/pull/1764#discussion_r1172670466, right?